assets/app/view/game/actionable.rb

```diff
@@ -86,6 +86,7 @@ def process_action(action)
         if hotseat
           @game_data[:turn] = game.turn
           @game_data[:round] = game.round.name
+          @game_data[:round_num] = game.round.round_num
```

Comment:
what about for non hot seat??

Reply:
Doesn't line 102 handle that? I looked where turn and round was stored, and only found these lines and line 101+103.